lib/api_endpoint/oas3/api_openapi_parameter_handler.py

```diff
@@ -71,7 +73,7 @@ def wrap_body_params(
         """
         # todo:
         # not unique enough. make it unique
-        wrapper_name = service_name + '_' + operation_name
+        wrapper_name = utils.get_str_camel_case(service_name + '_' + operation_name,  ".", "-", "_")
```

Comment:
As I expected (commented above), I see this sequence of three characters way too often - declare them once and reuse (and they might actually be 2, w/out the "-")

Reply:
Added constant and reused (answered above)